src/main/java/io/jenkins/plugins/audit/listeners/UserLogListener.java

```diff
@@ -93,19 +96,20 @@ protected void loggedIn(@Nonnull String username) {
      *
      * @param username name or ID of the user who logged out.
      */
-     @Override
-     protected void loggedOut(@Nonnull String username) {
-         String currentTime = Instant.now().toString();
-         Logout logout = LogEventFactory.getEvent(Logout.class);
+    @Override
+    protected void loggedOut(@Nonnull String username) {
+        String currentTime = Instant.now().toString();
+        Logout logout = LogEventFactory.getEvent(Logout.class);
 
-         RequestContext.setNodeName("master");
-         RequestContext.getRequestId();
-         RequestContext.setIpAddress(Stapler.getCurrentRequest().getRemoteAddr());
-         logout.setUserId(username);
-         logout.setTimestamp(currentTime);
-         logout.logEvent();
-         RequestContext.clear();
-     }
+        RequestContext.setNodeName("master");
+        RequestContext.getRequestId();
+        RequestContext.setIpAddress(Stapler.getCurrentRequest().getRemoteAddr());
+        RequestContext.setUserId(username);
+        logout.setUserId(username);
```

Comment:
So now I'm kind of wondering if we really need a userId parameter for a Login/Logout event if the userId is already a common attribute.